<?xml version="1.0" ?>
<launch>

	<arg name="smooth" default="false"/>

	<node name="control_trajectory_execution_action_server" type="control_trajectory_execution_action_server" pkg="control_trajectory_execution" output="screen" respawn="false">
		<rosparam file="$(find control_trajectory_execution)/config/control_trajectory_execution_action_server.yaml" command="load"/>
		<param name="smooth" value="$(arg smooth)"/>
	</node>

</launch>
